otj-metrics-actuate/src/main/java/com/opentable/metrics/actuate/micrometer/OtMicrometerToDropWizardExportConfiguration.java

```diff
@@ -0,0 +1,118 @@
+package com.opentable.metrics.actuate.micrometer;
+
+import java.text.Normalizer;
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Pattern;
+
+import com.codahale.metrics.MetricRegistry;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import io.micrometer.core.instrument.Clock;
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.config.MeterFilter;
+import io.micrometer.core.instrument.config.NamingConvention;
+import io.micrometer.core.instrument.dropwizard.DropwizardConfig;
+import io.micrometer.core.instrument.dropwizard.DropwizardMeterRegistry;
+import io.micrometer.core.instrument.util.HierarchicalNameMapper;
+import io.micrometer.core.lang.NonNull;
+import io.micrometer.core.lang.Nullable;
+
+/**
+ * Configures  {@link DropwizardMeterRegistry} with custom {@link HierarchicalNameMapper} and
+ * {@link HierarchicalNameMapper} to report micrometer metrics in the DropWizard infrastructure
+ */
+@Configuration
+public class OtMicrometerToDropWizardExportConfiguration {
+
+    private static final Logger log = LoggerFactory.getLogger(OtMicrometerToDropWizardExportConfiguration.class);
+    private static final Pattern blacklistedChars = Pattern.compile("[{}(),=\\[\\]/]");
+
+    private DropwizardConfig dropwizardConfig() {
+        return new DropwizardConfig() {
+            @Override
+            @NonNull
+            public String prefix() {
+                return "management.metrics.dw-new";
+            }
+
+            @Override
+            @Nullable
+            public String get(String s) {
+                return null;
+            }
+        };
+    }
+
+
+    private HierarchicalNameMapper hierarchicalNameMapper() {
+        return (id, convention) -> {
+            StringBuilder tags = new StringBuilder();
+            if (!"true".equals(id.getTag("absolute"))) {
+                for (Tag tag : id.getTags()) {
+                    tags.append(("." + /*convention.tagKey(tag.getKey()) + "."  + */ convention.tagValue(tag.getValue()))
+                        .replace(" ", "_"));
+                }
+                final String res = "v2." + id.getConventionName(convention) + tags;
```

Comment:
Why v2?

Reply:
puts all micrometer metrics there to avoid collision, hhowever potentially doubles metrics

Reply:
And easier to find. It is configurable now